package.json

```diff
@@ -21,11 +21,12 @@
     "eslint-config-google": "^0.9.1",
     "firebase-tools": "^7.2.2",
     "glob-all": "^3.1.0",
+    "har-validator": "^5.1.3",
```

Comment:
why was this added? please remove if it was added by accident

Reply:
That was no accident but as fix as suggested in issue #66 by @r0b78